### PR TITLE
chore: Added @rimolive, @hbelmiro and @DharmitD as approvers and reviewers for GitHub Actions

### PR DESCRIPTION
**Pull request by hbelmiro**
This PR adds @rimolive, @hbelmiro and @DharmitD as approvers and reviewers for GitHub Actions.
They have been working on [migrating tests from Prow to GitHub Actions](https://github.com/kubeflow/pipelines/issues/10744).

PRs:
- https://github.com/kubeflow/pipelines/pull/10887
- https://github.com/kubeflow/pipelines/pull/10924
- https://github.com/kubeflow/pipelines/pull/10926
- https://github.com/kubeflow/pipelines/pull/10932
- https://github.com/kubeflow/pipelines/pull/10946
- https://github.com/kubeflow/pipelines/pull/10961
- https://github.com/kubeflow/pipelines/pull/10923
- https://github.com/kubeflow/pipelines/pull/10871
- https://github.com/kubeflow/pipelines/pull/10842
- https://github.com/kubeflow/pipelines/pull/10930
- https://github.com/kubeflow/pipelines/pull/10702

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by hbelmiro** [comment]
cc @chensun @zijianjoy @james-jwu @rimolive @DharmitD

File: .github/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - chensun
+  - IronPan
+  - james-jwu
+  - zijianjoy
+  - rimolive
+  - hbelmiro
+  - DharmitD
```
Comment (DharmitD) [inline .github/OWNERS:8]: nitpick: Fix EoF

File: scripts/OWNERS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - chensun
+  - IronPan
+  - james-jwu
+  - zijianjoy
+  - rimolive
+  - hbelmiro
+  - DharmitD
```
Comment (DharmitD) [inline scripts/OWNERS:8]: nitpick: Fix EoF

**Review by DharmitD — COMMENTED** [review]
minor nitpick but otherwise,
/lgtm

@hbelmiro thank you for making this PR on our behalf! :)

File: .github/OWNERS (hunk 1 above)
Comment (hbelmiro) [reply .github/OWNERS]: Fixed.

File: scripts/OWNERS (hunk 2 above)
Comment (hbelmiro) [reply scripts/OWNERS]: Fixed.

**Comment by rimolive** [comment]
/lgtm

cc @chensun @zijianjoy @james-jwu

**Review by chensun — APPROVED** [review]
/lgtm
/approve

Thank you all for your contribution on migration the tests!

**Pull request merged**